editor/src/core/shared/optional-utils.ts

```diff
@@ -23,22 +23,22 @@ export function forceNotNull<T>(description: string, value: T | null | undefined
 }
 
 // FIXME This shouldn't be converting null into undefined
-export function optionalMap<T, U>(fn: (t: T) => U, t: T | null | undefined): U | null {
+export function optionalMap<T, U>(fn: (param: T) => U, t: T | null | undefined): U | null {
```

Comment:
`t` isn't allowed?

Reply:
I think it's the new eslint rule that thinks this t was shadowing the other t, which sounds silly?

Reply:
it is silly given this is a type def, but whatever. maybe there's some good reason.